update-ngxblocker

```diff
@@ -272,16 +280,23 @@ send_email() {
 		print_message "${BOLDYELLOW}WARN${RESET}: missing mail command => ${BOLDWHITE}disabling emails${RESET}.\n\n"
 	fi
 }
+send_email_via_mailgun() {
+	echo "Mailgunning report to: ${BOLDWHITE}$EMAIL${RESET}\n\n";
+	sed -i 's/\x1b\[[0-9;]*m//g' $EMAIL_REPORT
+	report="$(cat $EMAIL_REPORT)"
+	curl -s --user api:$MG_API_KEY https://api.mailgun.net/v3/$MG_DOMAIN/messages -F from='botblocker <botblocker@viti.io>' -F to=$EMAIL -F subject='Nginx Bad Bot Blocker Updated' -F text="$report"
+}
 
 get_options() {
 	local arg= opts=
 
-	while getopts :c:b:i:r:e:m:lnovqh opts "$@"
+	while getopts :c:b:i:r:e:g:a:d:m:lnovqh opts "$@"
```

Comment:
Add option for Mailgun FROM address

`while getopts :c:b:i:r:e:g:a:d:f:m:lnovqh opts "$@"`

Reply:
yup.